backend/src/main/kotlin/cz/loono/backend/api/controller/HealthcareProvidersController.kt

```diff
@@ -16,20 +22,23 @@ class HealthcareProvidersController {
     private lateinit var healthCareProvidersService: HealthcareProvidersService
 
     @GetMapping(value = ["$DOCTORS_PATH/update"])
-    fun updateData(): UpdateStatusMessage {
+    fun updateData(): UpdateStatusMessageDto {
         return healthCareProvidersService.updateData()
     }
 
-    @GetMapping(value = ["$DOCTORS_PATH/all"])
-    fun getAll(): String {
-        // TODO returns all providers in simple form
-        return "{}"
+    @GetMapping(value = ["$DOCTORS_PATH/all"], produces = [MediaType.APPLICATION_OCTET_STREAM_VALUE])
+    fun getAll(response: HttpServletResponse): ByteArray {
+        response.setHeader("Content-Disposition", "attachment; filename=providers.zip")
```

Comment:
Čisto filozofická otázka - nechceme nejaký iný názov? Napr. providers-2021-10 ? Viem, komplikuje to situáciu, preto filozofujem. Prípadne HealthCareProviders.zip?

Reply:
Good point - dam tam nejaky timestamp.

Reply:
Mozna bych mohl posledni update ukladat a pouzit v nazvu souboru.